package.json

```diff
@@ -32,7 +32,14 @@
       ]
     },
     "views": {
-      "ONE-Views": []
+      "ONE-Views": [
+        {
+          "id": "CompilerView",
```

Comment:
Which name would be better among `Compiler` and `CompilationEnv`?
I guess we should use one name through entire project. :-)

Reply:
IHMO, how about `CompilationView`? AFAIU, the `CompilationEnv` could be multiple on and be an env instance itself ONE-vscode.

Reply:
/cc @Samsung/one-vscode